javascript-carousel/main.js

```diff
@@ -0,0 +1,84 @@
+const images = ['001', '004', '007', '025', '039'];
```

Comment:
By doing this, you're hard coding the image values. We want to avoid this, because it prevents this from being truly dynamic. We should be able to only add an element to HTML and the JavaScript should always work.